project-progression-policy.md

```diff
@@ -180,7 +181,6 @@ To graduate from Sandbox or Incubation status, or for a new project to join as a
 
  * Have a defined governing body of at least 5 or more members (owners and core maintainers, or similar technical role), of which no more than 1/3 is affiliated with the same employer. In the case there are 5 governing members, 2 may be from the same employer.
 * Have a healthy number of committers from at least two organizations. A committer is defined as someone with the commit bit; i.e., someone who can accept contributions to some or all of the project.
-* Adopt a Code of Conduct.
```

Comment:
Rather than deleting it, we should move this to an earlier stage.   Incubation at latest, and we can debate whether Sandbox too.